### ISSUE TITLE
Only the first element of the array whitelisted in media-feature-name-whitelist/blacklist

### ISSUE THREAD

**Issue by bsrdjan** [stylelint/stylelint#2536]
When array of future names defined for the `media-feature-name-whitelist` rule, only the first feature name of the array is whitelisted and errors reported for others.

CSS to reproduce:

```css
media only screen and (max-width: 1200px) {
  .ui-container {
    max-width: 1200px;
    margin: 0 auto;
  }
}

@media only screen and (min-width: 600px) {
  .ui-container {
    max-width: 600px;
    margin: 0 auto;
  }
}
```

stylelint configuration to reproduce:

```js
module.exports = {
  "extends": "stylelint-config-standard",

  "rules": {
    "media-feature-name-whitelist": [ "max-width", "min-width" ]
  }
};
```

stylelint version: 7.10.1, tested from the command line:

```shell
stylelint test.css

test.css
 8:25  ✖  Unexpected media feature name "min-width"   media-feature-name-whitelist
```

**Comment by jeddy3** [comment]
@bsrdjan Thanks for the report. 

Looks like we're missing a `rule.primaryOptionArray = true`.

**Comment by evilebottnawi** [comment]
@davidtheclark can we add ability in tests multiple error:
```shell
    code: '.a { property: value; property-two: value }',
    errors: [{
        message: messages.rejected("property"),
        line: 1,
        column: 7
    }, {
        message: messages.rejected("property-two"),
        line: 1,
        column: 19
    }]
  },
```

It is allow testing similar error :smile:

**Comment by evilebottnawi** [comment]
be good add ability to tests this and then i can do PR for fix bug and add tests for `*-whitelist/blacklist` and maybe other rules

**Comment by davidtheclark** [comment]
@evilebottnawi: This was recently discussed with @hudochenkov. I think we realized that we could test this simply by passing an array config that would fail if `rule.primaryOptionArray !== true`.

**Comment by evilebottnawi** [comment]
@davidtheclark good, need do it :smile:

**Comment by jeddy3** [comment]
> @davidtheclark good, need do it 😄

My bad, I missed `media-feature-name-*` [first time around](https://github.com/stylelint/stylelint/pull/2466/files).

**Comment by evilebottnawi** [comment]
/cc @davidtheclark we can fix this now and first

>  I think we realized that we could test this simply by passing an array config that would fail if

and then fix issue?

**Comment by davidtheclark** [comment]
@evilebottnawi I don't think I understand your question. Any order you want to do things is fine by me.

**Comment by hudochenkov** [comment]
Done in #2632.
